assessments/intake-process.md

```diff
@@ -0,0 +1,45 @@
+# Security Assessment Priorities & Pipeline Intake Process
+
+SIG-Security has a volunteer team of subject matter experts and industry
+professionals dedicated to helping SIG-Security members, the TOC, and the larger
+CNCF community maintain an understanding of the current state of security in the
+cloud native ecosystem and helping cloud native projects succeed.
+
+The following process describes how projects are prioritized for security
+assessments.
+
+# Authority
+
+* The [Security Assessment
+  Facilitator](../governance/roles.md#security-assessment-facilitator) maintains
+  the Assessment queue
+* A named chair provides oversight for the Security Assessment initiative. This
+  chair can adjust queue, and is also responsible for conveying TOC
+  instructions, when needed. (Sarah Allen is the current named Chair for
+  Security Assessments.)
+
+# Pre-conditions
+
+* The project is either a CNCF project OR an assertion that the project is cloud
+  native (any objection must be resolved before an assessment would be
+  considered)
+* The project has identified a project lead and has a written self-assessment
+
+# Intake priorities
+
+1. TOC requests SIG-Security review a specific project or adjust priorities a.
```

Comment:
perhaps make this as a sub bullet

1. abc
  a. def
  b. ghi
2. aaa

Reply:
done!